pkgs/top-level/all-packages.nix

```diff
@@ -4840,8 +4840,9 @@ with pkgs;
   };
 
   geekbench4 = callPackage ../tools/misc/geekbench/4.nix { };
-  geekbench5 = callPackage ../tools/misc/geekbench { };
-  geekbench = geekbench5;
+  geekbench5 = callPackage ../tools/misc/geekbench/5.nix { };
+  geekbench_6 = callPackage ../tools/misc/geekbench/6.nix { };
```

Comment:
Note: At top-level, package name and version should be split by an underscore _. I'm not changing previous packages because I don't want to affect any use legacy usage. This will naturally be sorted out when these legacy packages are removed. So no need to bother about this.

Reply:
In this case, it is better to add aliases for the old names and fix the issue once for all.